buildSrc/src/main/kotlin/brd/Libs.kt

```diff
@@ -24,7 +24,7 @@
  */
 package brd
 
-const val KOTLIN_VERSION = "1.4.31"
+const val KOTLIN_VERSION = "1.5.0"
```

Comment:
I had to upgrade kotlin version to 1.5.0 this might require a regression test

https://github.com/checkout/frames-android/issues/95